frontend/docsettings.lua

```diff
@@ -481,7 +539,7 @@ end
 --- Returns path to book custom metadata file if it exists, or nil.
 function DocSettings:getCustomMetadataFile(doc_path)
     doc_path = doc_path or self.data.doc_path
-    for _, mode in ipairs({"doc", "dir"}) do
+    for _, mode in ipairs({"doc", "dir", "hash"}) do
```

Comment:
No check for `document_metadata_hash_enabled`.

Reply:
Updated, I'll let you resolve since I did it too early before.